src/compiler/interface.jl

```diff
@@ -50,9 +50,22 @@ sensitivity(y) = error("Output should be scalar; gradients are not defined for o
     gradient(f, args...)
 
 Returns a tuple containing `∂f/∂x` for each argument `x`,
-the derivative (for scalar x) or the gradient.
+the derivative (for scalar `x`) or the gradient.
 
 `f(args...)` must be a real number, see [`jacobian`](@ref) for array output.
+
+See also [`withgradient`](@ref) to keep the value `f(args...)`,
+and `pullback`](@ref) for value and back-propagator.
```

Comment:
```suggestion
and [`pullback`](@ref) for value and back-propagator.
```

Reply:
We can see what CI thinks, I didn't think there was a doc entry for `pullback` so it might complain.

Reply:
Turns out it has no complaints about this, but does have a few similar complaints... although they don't cause a failure so that's OK.

https://github.com/FluxML/Zygote.jl/pull/1010/checks#step:6:89